app/src/main/java/org/eyeseetea/malariacare/data/database/iomodules/dhis/exporter/PushController.java

```diff
@@ -51,6 +52,11 @@ public PushController(Context context) {
     }
 
     public void push(final IPushControllerCallback callback) {
+        boolean isNetworkAvailable = AUtils.isNetworkAvailable();
+
+        if (!isNetworkAvailable) {
```

Comment:
@idelcano The network validation was being performed in the data sources, UserAuthenticationDataSource, PushDhisSDKDataSource, PullDhisSDKDataSource. Is there any reason to move this validation for push to controller?